giranezafiacre/politico
# all api except optional

> #### What does this PR do?
> it has UI and server
>
> #### Description of Task to be completed
> UI and APIs
>
> #### How should this be manually tested?
>npm install
>npm i joi
>npm i .dotenv
>and other dependencies
> #### Any background context you want to provide?
>none
> #### What are the relevant pivotal tracker stories?
>user can signup,can signin and can vote,can see politician
>politician can express interest to run for candidate
> #### Screenshots (if appropriate)
> none
